jlm/util/Program.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2025 Nico Reißmann <nico.reissmann@gmail.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include <jlm/util/common.hpp>
+#include <jlm/util/Program.hpp>
+#include <jlm/util/strfmt.hpp>
+
+namespace jlm::util
+{
+
+std::filesystem::path
+tryFindExecutablePath(const std::string_view & programName)
```

Comment:
Const reference to string view should just be a string view by value

Reply:
Fixed.